epub3-pub-utils/src/main/resources/xml/xproc/create-metadata.merge.xsl

```diff
@@ -2,7 +2,6 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0"
     xmlns="http://www.idpf.org/2007/opf" xmlns:dc="http://purl.org/dc/elements/1.1/"
     xmlns:f="http://www.daisy.org/ns/pipeline/internal-functions"
-    xmlns:d="http://www.daisy.org/ns/pipeline/data"
     xmlns:xs="http://www.w3.org/2001/XMLSchema" exclude-result-prefixes="#all"
```

Comment:
I've never properly understood how namespaces work in XSLT. Maybe this is a bug in Saxon?